web/gds-admin-ui/src/utils/index.js

```diff
@@ -62,7 +62,29 @@ function getCookie(name = '') {
     return '';
 }
 
+function getStatusClassName(status = '') {
+    switch (status) {
+        case Status.VERIFIED:
+            return 'bg-success'
+        case Status.SUBMITTED:
+            return 'bg-secondary'
+        case Status.PENDING_REVIEW:
+        case Status.EMAIL_VERIFIED:
+            return 'bg-warning'
+        case Status.ERRORED:
+        case Status.REJECTED:
+            return 'bg-danger'
+        case Status.APPEALED:
+            return 'bg-primary'
+        case Status.REVIEWED:
+        case Status.ISSUING_CERTIFICATE:
+            return 'bg-info'
+        default:
+            return undefined
+    }
+}
```

Comment:
I suspect that we may want to use this function in other places as well since color to status mappings will happen frequently. That might mean that in the future we'll want to remove the `bg-` prefix (so that we can use `text-` prefix, for example); but I don't think we need to do that now, just wanted to call it out. 

Nice one!